package/gluon-core/check_site.lua

```diff
@@ -44,8 +44,10 @@ for _, config in ipairs({'wifi24', 'wifi5'}) do
 				120, 122, 124, 126, 128, 132, 134, 136, 138, 140, 142, 144,
 				149, 151, 153, 155, 157, 159, 161, 165, 169, 173 }
 			need_one_of({config, 'channel'}, channels)
-			need_chanlist({config, 'outdoor_chanlist'}, channels, false)
-			need_one_of({config, 'outdoors'}, {true, false, 'preset'}, false)
+
+			local outdoors = need_one_of({config, 'outdoors'}, {true, false, 'preset'}, false)
+			local outdoor_chanlist_required = (outdoors == "preset" or outdoors == true)
```

Comment:
```suggestion
			local outdoor_chanlist_required = (outdoors ~= false)
```
`outdoors` defaults to 'preset', which makes sense - we want communities to support outdoor mode to respect regulatory rules.